File: 编程有关/python/作业/ex1_triangle.py
#coding=utf-8
from math import sqrt,acos
mc = ['一','二','三']
a = [0] * 3
p = 0
for i in range(3):
    a[i] = float(input(f'请输入第{mc[i]}条边边长：'))
    p += a[i]
p /= 2
print(f'三角形的面积为：{round(sqrt(p * (p - a[0]) * (p - a[1]) * (p - a[2])),1)}')
print(f'角A为{round(acos((a[0] ** 2 + a[1] ** 2 - a[2] ** 2) / (2 * a[0] * a[1])),2)}弧度')

File: 编程有关/程序设计基础（C语言）/作业（高建良）/ex1_triangle.c
#include <stdio.h>
#include <math.h>
double a,b,c,p;
int main()
{
	printf("请输入第一条边边长:");
	scanf("%lf",&a);
	printf("请输入第二条边边长:");
	scanf("%lf",&b);
	printf("请输入第三条边边长:");
	scanf("%lf",&c);
	p = (a + b + c) / 2;
	printf("三角形的面积为:%.1lf\n",sqrt(p * (p - a) * (p - b) * (p - c)));
	printf("角A为%.2lf弧度",acos((b*b + c*c - a*a)/(2 * b * c)));
	return 0;
}
